_posts/2019-08-05-cognito-azuread-sso.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "Securing your web applications in AWS using SSO with Azure Active Directory"
+date:   2019-08-05
+tags: aws cognito oidc saml2 sso azure
+author: Paul Symons
+---
+
+It's a fairly common scenario where you want to secure a web application in the cloud; but with so many technologies and different ways to secure your applications, it can be difficult to know where to get started. You may have questions, such as:
+
+- should I handle authentication within my web application itself, or within infrastructure?
+- how can I integrate with my organisation's identity provider?
+- can I automate the securing of my web application?
+- how can others in my organisation leverage this pattern?
+
+At [Mechanical Rock](https://www.mechanicalrock.io/our-expertise), two of our pillars of excellence are
+* [Cloud Native Applications](https://mechanicalrock.io/our-expertise/cloud-native-applications), and 
+* [Enterprise DevOps](https://mechanicalrock.io/our-expertise/enterprise-devops)
+
+ We leverage _best of breed_ cloud native and managed services to build high quality, cutting edge solutions. Delivering these solutions using Infrastructure-As-Code (IAC) is not simply part of our commitment to architecting steadfast **continuous integration delivery** workflows; it underpins the way we collaborate and share learnings with the client teams we engage with.
```

Comment:
`continuous integration delivery` ?  continuous delibery or continuous integration - I wouldn't put both in the same sentence unless you're trying to invest a new term.  If you are...don't! :)

Reply:
i think i meant to actually put `continuous integration and delivery`, but perhaps better to just write it as `continuous integration`